### DISCUSSION TITLE
Contributing to Ubicloud

### DISCUSSION THREAD

**Discussion by Haruna2004** [ubicloud/ubicloud#1487] [General]
Dear Ubicloud Team
I'm interested in supporting Ubicloud with my skills and experience but I face the issue of figuring out how I can be useful to the project.

My background: I have 3 years of experience building full-stack applications using Next and Nodejs.
Six months ago, I transitioned from a Software Engineer role at Beebatics to building cloud computing infrastructure in Africa.

My goal is to build a cloud deployment-as-a-service platform that automates DevOps processes, similar to [Render](https://render.com), but running on bare metal servers. I've developed two prototypes: one automates containarizing a project from source code and deploys the container to a local Kubernetes cluster. The second service takes a frontend source code URL (React, Next.js, Vite, etc.) then builds and deploys it to an object storage.  A request handler  allows the site to be accessed from a subdomain example (project-id).onhcp.com

While focusing on cloud computing, I built [Storepoint.ng](https://www.storepoint.ng/) to win a hackathon and fund my project.
I'm also about to interview for a Software Engineering (Python & K8s) role at Canonnical.

I'm very interested in contributing to Ubicloud. Please let me know if there are any opportunities where my skills and experience can be of value.

**Comment by ozgune** [comment]
Hey @Haruna2004, thank you for your interest.

I'm delayed in replying to your question. At this stage, our hiring process is mostly outbound and we don't have a formal process set up for inbound applications. Once we have a formal process around inbound applications, we're going to provide a link on our website accordingly.
